src/sheet/part-item.c

```diff
@@ -274,7 +274,6 @@ PartItem *part_item_canvas_new (Sheet *sheet, Part *part)
 	    oregano_options_debug_boxes () ? GOO_CANVAS_ITEM_VISIBLE : GOO_CANVAS_ITEM_INVISIBLE, NULL);
 
 	priv->label_group = goo_canvas_group_new (goo_item, "width", -1.0, "height", -1.0, NULL);
-	g_object_unref (goo_item); // FIXME wtf? why?
 
```

Comment:
This is wrong https://developer.gnome.org/goocanvas/stable/GooCanvasGroup.html#goo-canvas-group-new

Reply:
In the original code it is stated that unreferencing the item is wrong (see the FIXME comment) !

This is consistent with the documentation that you quoted above: "... the item will automatically be freed when it is removed from the parent." !!

Reply:
In other words, the original code creates a new group with a given parent and then immediately destroys the parent (which also destroys the group, as explained in the documentation link that you quoted).

What's the sense of that ??! It sounds insane to me and it was sounding insane to the original author too !...

Reply:
I have just completed testing it with the latest ngspice which has been published a few hours ago and everything works fine.

I would liketo move on the above issue, are you experiencing crashes or other unintended behavior ?? I have not heard anything back from you for a few days... Please let me know, thanks.